### PR TITLE
community: add gpt-4-0125-preview model prices

### PR DESCRIPTION
**Pull request by orarbel**
- **Description:** Adding for callbacks to calculate the cost when using gpt-4-0125-preview model. 

https://openai.com/pricing

### CODE REVIEW

File: libs/community/langchain_community/callbacks/openai_info.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "gpt-4-32k-0613": 0.06,
     "gpt-4-vision-preview": 0.01,
     "gpt-4-1106-preview": 0.01,
+    "gpt-4-0125-preview": 0.01,
```
Comment (huanvo88) [inline libs/community/langchain_community/callbacks/openai_info.py:18]: Can you also add `gpt-4-turbo-preview`, which always points to the latest `gpt-4-turbo` model

File: libs/community/langchain_community/callbacks/openai_info.py [hunk 2]
```diff
@@ -24,6 +25,7 @@
     "gpt-4-32k-0613-completion": 0.12,
     "gpt-4-vision-preview-completion": 0.03,
     "gpt-4-1106-preview-completion": 0.03,
+    "gpt-4-0125-preview-completion": 0.03,
```
Comment (huanvo88) [inline libs/community/langchain_community/callbacks/openai_info.py:28]: Can you also add `gpt-4-turbo-preview-completion`, which always points to the latest gpt-4-turbo model

**Comment by hwchase17** [comment]
thanks, was added in https://github.com/langchain-ai/langchain/pull/17349

**Comment by ricoms** [comment]
https://github.com/langchain-ai/langchain/pull/16774 thanks for duplicating

**Pull request closed without merging**